braid/_braid_status.c

```diff
@@ -148,6 +150,42 @@ braid_StatusGetDone(braid_Status status,
    return _braid_error_flag;
 }
 
+braid_Int
+braid_StatusGetTIUL(braid_Status status,
```

Comment:
Shouldn't level_ptr be a regular int, and not a pointer?  Its not an input, right, it's just a level index?  Same goes for the next function.

Reply:
To do this we should change (or add) the ACCESSOR_HEADER_GET macros. Currently they all force a pointer type. I think this would require a new macro that allows mixed input/output:

```
#define ACCESSOR_HEADER_GET3(stype,param,vtype1,vtype2,vtype3) \
braid_Int braid_##stype##StatusGet##param(braid_##stype##Status s, braid_##vtype1 *v1, braid_##vtype2 *v2, braid_##vtype3 v3);
```
where the first 2 arguments are pointers and the 3rd argument is a plain old value. Should I implement this? I'll have to give it a different name than ACCESSOR_HEADER_GET3.

Reply:
I ended up calling it ACCESSOR_HEADER_GET1_IN2 etc. for get 1 value with 2 inputs. See commits 5b12ba731b8748734121727c96a5f51eadc5de54 and b8d6e6f6d7a58c2eb9afe0aee01945aad5832cb8